funcx_endpoint/funcx_endpoint/cli.py

```diff
@@ -169,6 +171,28 @@ def start_endpoint(*, name: str, endpoint_uuid: str | None):
     )
 
 
+@app.command(name="logout", help="Logout from all endpoints")
+@click.option("--environment", default=None, help="The environment to logout from")
+@common_options
+def logout_endpoint(environment: str | None):
+    """
+    Logout from all endpoints and remove cached authentication credentials
+
+    TODO Currently there does not seem to be credentials associated with
+     specific endpoints, so will just have to remove all tokens for
+     all environments using the default LoginManager.
```

Comment:
This is a good observation, but leaving it as a `TODO` ~comment~ method documentation in the code means that it won't (practically) ever be on our team radar.  Suggest:
1. Create a proper ticket for this -- this is our team visibility to actually prioritize and move on it.
1. Removing mention of this in code, OR
1. Leave mention of it, but link to the ticket.